src/components/home.css

```diff
@@ -1,39 +1,85 @@
+/* page container */
+.page-container {
+    background-color: #24323F
+}
+
 /* movie container */
 .grid-container {
     display: grid;
-    justify-content: space-evenly;
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-    grid-gap: 10;
-    background-color: #113049;
-    padding: 10px;
-    overflow: hidden;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    grid-column-gap: 1.25em;
+    grid-row-gap: 2.5em;
+    padding-left: 5em;
+    padding-right: 5em;
+    padding-bottom: 1.25em;
+} 
+h1{
+    margin-top: 0;
+    padding-top: 3.5em;
+    padding-left: 2.5em;
+    text-align: left;
+    color: #FDFEFF;
+}
+hr {
+    margin-top: 2.5em;
+    margin-right: 5em;
+    margin-bottom: 3.125em;
+    margin-left: 5em;
+    color: #303E4B;
 }
-
-
 
 /* movie card */
 
-.grid-item {
+.image-card {
     position: relative;
-    text-align: center; 
-    padding: 30px; 
-    font-size: 30px;     
 }
-
 .movie-card-top {
-    height: 500px;
     border-radius: 5px;
+    max-width: 100%;
+}
+.rating {
+    width: 2.8125em;
+    height: 2.8125em;
+    position: absolute;
+    top: 1.25em;
+    right: 1.25em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    background-color: #74D5AA;
+    color: #294448;  
 }
-
 .movie-release-date {
+    color: #666B73;
+    padding-top: 1.25em;
     text-align: left;
-    color: #5A5F61;
-    font-size: 16px;
-    padding-top: 20px;
 }
-
 .movie-title {
     text-align: left;
-    color: #AFBABA; 
-    font-size: 20px;
+    color: #FAFAFC; 
+    font-size: 1.25em;
+}
+
+/* Design for tablets and phones */
+/* Always add media queries at the bottom of the css file to avoid overriding issues */
+@media only screen and (max-width: 768px) {
```

Comment:
I would move to three column layout bit earlier because this looks very tight to me :D

![image](https://user-images.githubusercontent.com/12229968/51784735-606a6400-2156-11e9-8bf8-54857632a504.png)